include/odp/api/spec/ipsec.h

```diff
@@ -455,6 +458,16 @@ typedef struct odp_ipsec_config_t {
 	 */
 	odp_bool_t stats_en;
 
+	/**
+	 * Packet vector configuration for async and inline operations
```

Comment:
>inline operations

For inline, having this config here will conflict with the config in odp_pktin_queue_param_t right ?
I think the one in odp_pktin_queue_param_t together with odp_pktio_config_t::inbound_ipsec should suffice for 
IPSec inline vector support ?

Reply:
Are you thinking about the case where inline processed inbound packets are delivered to the application through the pktin queue? The current API does not allow it as the packets are always delivered through the IPsec destination queues. Even if delivery through pktin was supported, I do not see a conflict since the API text here says that the config applies to the IPsec destination queues and the default queue.

Are you still planning to propose an API change that would allow an implementation to deliver inbound inline completion packets through pktin queues?

Reply:
I believe in our case, such a spec change is must as inline decrypted packets will be treated as plain packets and will reach PKTIO PKTIN queue or specific plain packet matching CoS but not destination queue provided in IPSec SA.